src/render/components/Dom.jsx

```diff
@@ -33,10 +33,9 @@ const DOM = props => {
 		appMarkup = '',
 		assetPublicPath,
 		baseUrl,
-		clientFilename,
 		head,
 		initialState = {},
-		scripts = [],
+		scripts,
```

Comment:
If scripts aren't passed I see we would get prop errors but would this _maybe_ cause server errors of empty?